README.md

```diff
@@ -194,30 +195,33 @@ For most users, setting `image: [path-to-image]` on a per-page basis should be e
 * `path` - The relative path to the image. Same as `image: [path-to-image]`
 * `twitter` - The relative path to a Twitter-specific image.
 * `facebook` - The relative path to a Facebook-specific image.
-* `height` - The height of the Facebook (`og:image`) image and JSON-LD image object.
-* `width` - The width of the Facebook (`og:image`) image and JSON-LD image object.
+* `height` - The height of the image in pixels.
+* `width` - The width of image in pixels.
 
 The JSON-LD will default to the `image: ` tag unless `path: ` `height: ` and `width: `.
 
 You can use any of the above, optional properties, like so:
 
 ```yml
-image:
-  path: /img/banner.png
-  twitter: /img/twitter.png
-  facebook: /img/facebook.png
-  height: 100
-  width: 100
+image: /img/banner.png
 ```
 
-Or if you have no site specific images simply:
+Or if you have Facebook or Twitter specific images simply use:
 
 ```yml
 image:
-  path: /img/banner.png
+  default:
+    path: /img/banner.png
+    height: 100
+    width: 100
+  facebook:
+    path: /img/facebook.png
+    height: 90
+    width: 90
+  twitter: /img/twitter.png
```

Comment:
I spent most of the day thinking about this and how to handle this, but I feel that I need to propose a change on how images are listed.

The current configuration (below) has problems.

```yml
image:
  twitter: /img/twitter.png
  facebook: /img/facebook.png
  height: 100
  width: 100
```

* It is not clear from the configuration what the height and width are for (The documentation says that the height and width are set for the Facebook image, but we can't infer from the config.)
* If we want to add another image that requires dimensions (i.e. a non-network specific image into the JSON-LD), we must cobble together either a different named field for height and width or create nested fields. This is because there is no guarantee that the two images would have the same dimensions. It would actually make more sense if they had different dimensions, hence the need for a network specific image.

My proposal is: 

```yml
image:
  default:
    path: /img/banner.png
    height: 100
    width: 100
  facebook:
    path: /img/facebook.png
    height: 90
    width: 90
  twitter: /img/twitter.png
```

The way I have it set up, the following will still work:

```yml
image:
  default:
    path: /img/banner.png
  facebook:
    path: /img/facebook.png
  twitter: /img/twitter.png
```

and this will also work.

```yml
image:
  default: /img/banner.png
  facebook:/img/facebook.png
  twitter: /img/twitter.png
```

and this:

```yml
image: /img/banner.png
```

The downside is that **this will break current implementation** of those who use the `height` and `width` fields in the original for Facebook images. So I understand if you reject this idea. 

Unless we decide to use some type of ugly handling... 

Thoughts?